misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/natives/test/StartServiceSkeleton.java

```diff
@@ -111,7 +110,7 @@ private void initTempDir(String sourceRoot) {
             try {
                 Files.createDirectories(projectRoot);
             } catch (IOException e) {
-                throw new BallerinaIOException(String.format("Service skeleton creation failed. Failed to create " +
+                throw new BallerinaException(String.format("Service skeleton creation failed. Failed to create " +
```

Comment:
We shouldn't be throwing `BallerinaException`. If this logic is somewhere deep down in the module I'd suggest introducing a separate exception type and then catch it at an outer level and throw `ErrorValue`